src/tutorial/src/step-2/description.md

```diff
@@ -1 +1,42 @@
-# Two-way Binding
+# Handling User Input
+
+Let's make our example more interactive! What if we could allow user to change `colorsNumber` from the UI? To implement this, we need to start with adding an `<input>` field to our component template
+
+```html{4}
+<div class="panel">
+  <h1>Guess the Color</h1>
+  <h2>Number of colors: 3</h2>
+  <input type="number">
+</div>
+```
+
+Now, we have to provide some initial value to our input, and we want to display `colorsNumber` there. Let's use a dynamic binding to provide a value:
+
+```html
+<input type="number" :value="colorsNumber" />
+```
+
+However, when we change the number in the input field, we don't see any changes on `Number of colors` in the template 🤔. It happens because we only have _one-way binding_ now, displaying reactive property in the input field but not _changing_ it on user input. Let's listen to the `input` DOM event on the `<input>` field and update `colorsNumber` on it:
```

Comment:
@skirtles-code yes, the idea of the tutorial is to let people code on REPL. I am not really sure how we could encourage them to code rather than read only 🤔

Reply:
Regarding separating `v-bind` and events + v-model: this sounds like a good idea 👍🏻

Reply:
Split lessons to bite-sized chunks